docs/api/role-management/get.asciidoc

```diff
@@ -0,0 +1,92 @@
+[[role-management-api-get]]
+===== Get Roles
+
+To retrieve all roles from {es}'s `native` realm, issue a GET request to the 
+`/api/security/role` endpoint:
+
+[source,js]
+--------------------------------------------------
+GET /api/security/role
+--------------------------------------------------
+// KIBANA
+
+A successful call returns a response code of `200` and a response body containing a JSON 
+representation of the roles.
+
+[source,js]
+--------------------------------------------------
+[
+  {
+    "name": "my_kibana_role",
+    "metadata" : {
+      "version" : 1
+    },
+    "transient_metadata": {
+      "enabled": true
+    },
+    "elasticsearch": {
+      "indices": [ ],
+      "cluster": [ ],
+      "run_as": [ ]
+    },
+    "kibana": [ {
+        "privileges": [ "all" ]
+    } ],
+  },
+  {
+    "name": "my_admin_role",
+    "metadata" : {
+      "version" : 1
+    },
+    "transient_metadata": {
+      "enabled": true
+    },
+    "elasticsearch": {
+      "cluster" : [ "all" ],
+      "indices" : [ {
+        "names" : [ "index1", "index2" ],
+        "privileges" : [ "all" ],
+        "field_security" : {
+          "grant" : [ "title", "body" ]
+        },
+        "query" : "{\"match\": {\"title\": \"foo\"}}"
+      } ],
+    },
+    "kibana": [ ]
+  }
+]
+--------------------------------------------------
+
+
+To retrieve a specific role from {es}'s `native` realm, issue a GET request to
+the `/api/security/role/<rolename>` endpoint:
+
+[source,js]
+--------------------------------------------------
+GET /api/security/role/my_kibana_role
+--------------------------------------------------
+// KIBANA
+
+A successful call returns a response code of `200` and a response body containing a JSON 
+representation of the role.
+
+[source,js]
+--------------------------------------------------
+{
+  "name": "my_kibana_role",
+  "metadata" : {
+    "version" : 1
+  },
+  "transient_metadata": {
+    "enabled": true
+  },
+  "elasticsearch": {
+    "cluster": [ ],
+    "indices": [ ],
+    "run_as": [ ]
+  },
+  "kibana": [ {
+      "privileges": [ "all" ]
+  } ],
+}
+--------------------------------------------------
```

Comment:
- Same edits as above, plus added separate headings for getting all Kibana roles and a specific role.
- Also, the edit below does not include the code, as noted by _put_code_here_

[[role-management-api-get]]
=== Get role
experimental[This API is *experimental* and may be changed or removed completely in a future release. Although the underlying mechanism of enforcing role-based access control is stable, the APIs for managing the roles are currently experimental.]

Retrieves all {kib} roles, or a specific role.

==== Get all {kib} roles

===== Request

To retrieve all roles, issue a GET request to the 
`/api/security/role` endpoint.

_put code here_

===== Response

A successful call returns a response code of `200` and a response body containing a JSON 
representation of the roles.

_put code here_

==== Get a specific role

===== Request
To retrieve a specific role, issue a GET request to
the `/api/security/role/<rolename>` endpoint.

_put code here_


===== Response

A successful call returns a response code of `200` and a response body containing a JSON 
representation of the role.

_put code here_